### ISSUE TITLE
Get-SFTPFile -NoProgress causes exception

### ISSUE THREAD

**Issue by petesweeps** [darkoperator/Posh-SSH#122]
PS C:\Users\DewbieDooDoo> Get-SFTPFile -NoProgress
Get-SFTPFile : Parameter set cannot be resolved using the specified named parameters.
At line:1 char:1
+ Get-SFTPFile -NoProgress
+ ~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Get-SFTPFile], ParameterBindingException
    + FullyQualifiedErrorId : AmbiguousParameterSet,SSH.GetSftpFile

The Get-SCPFile does not have the same issue:
PS C:\Users\DewbieDooDoo> Get-SCPFile -NoProgress

cmdlet Get-SCPFile at command pipeline position 1
Supply values for the following parameters:
ComputerName[0]:

**Comment by darkoperator** [comment]
Any other details like what version of PowerShell, version of Posh-SSH, Version of Windows ..etc ?

on Windows 10 with PS 5.1 and latest:
```
PS C:\Users\Carlos> Get-SCPFile -NoProgress -ComputerName 192.168.1.156 -Credential carlos -LocalFile .\bashhistory.txt -RemoteFile /home/carlos/.bash_history
PS C:\Users\Carlos>
```

**Comment by petesweeps** [comment]
This appears to only impact SFTPFile, the SCPFile works fine with the switch.

**From Windows 10, PS 5.1, and Latest Posh-SSH:**

PS C:\Users\DewbieDooDoo> get-host
 ```Name             : ConsoleHost
 Version          : 5.1.14393.103
 InstanceId       : 50e40d53-718e-4b87-8440-6d0b39f86886
 UI               : System.Management.Automation.Internal.Host.InternalHostUserInterface
 CurrentCulture   : en-US
 CurrentUICulture : en-US
 PrivateData      : Microsoft.PowerShell.ConsoleHost+ConsoleColorProxy
 DebuggerEnabled  : True
 IsRunspacePushed : False
 Runspace         : System.Management.Automation.Runspaces.LocalRunspace
```
PS C:\Users\DewbieDooDoo> $psversiontable
``` Name                           Value
 PSVersion                      5.1.14393.103
 PSEdition                      Desktop
 PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
 BuildVersion                   10.0.14393.103
 CLRVersion                     4.0.30319.42000
 WSManStackVersion              3.0
 PSRemotingProtocolVersion      2.3
 SerializationVersion           1.1.0.1
```
PS C:\Users\DewbieDooDoo> **Get-SFTPFile -noProgress**
```Get-SFTPFile : Parameter set cannot be resolved using the specified named parameters.
At line:1 char:1
+ Get-SFTPFile -noProgress
+ ~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Get-SFTPFile], ParameterBindingException
    + FullyQualifiedErrorId : AmbiguousParameterSet,SSH.GetSftpFile
```

**From Windows 2012 R2, PS 4.0, and Latest Posh-SSH:**

PS C:\Users\Administrator> get-host
```Name             : ConsoleHost
Version          : 4.0
InstanceId       : bce87148-7dda-45a8-883c-9671b90ba502
UI               : System.Management.Automation.Internal.Host.InternalHostUserInterface
CurrentCulture   : en-US
CurrentUICulture : en-US
PrivateData      : Microsoft.PowerShell.ConsoleHost+ConsoleColorProxy
IsRunspacePushed : False
Runspace         : System.Management.Automation.Runspaces.LocalRunspace
```
PS C:\Users\Administrator> $psversiontable
```Name                           Value
----                           -----
PSVersion                      4.0
WSManStackVersion              3.0
SerializationVersion           1.1.0.1
CLRVersion                     4.0.30319.34014
BuildVersion                   6.3.9600.17090
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0}
PSRemotingProtocolVersion      2.2
```
PS C:\Users\Administrator> **Get-SFTPFile -NoProgress**
```Get-SFTPFile : Parameter set cannot be resolved using the specified named parameters.
At line:1 char:1
+ Get-SFTPFile -NoProgress
+ ~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Get-SFTPFile], ParameterBindingException
    + FullyQualifiedErrorId : AmbiguousParameterSet,SSH.GetSftpFile
```

**Comment by darkoperator** [comment]
At airport in a family trip, will test when I get a chance before new years

**Comment by darkoperator** [comment]
Should be fixed now in version 1.7.7, fixed it on the plane ride to my vacation and pushed it out this morning when I found some internet access. Test and let me know if it worked out for you.

**Comment by petesweeps** [comment]
Thanks! It works great.
